pages/common/git-cp.md

```diff
@@ -0,0 +1,12 @@
+# git cp
+
+> Copy an existing file. A part of git-extras.
+> More information: <https://github.com/tj/git-extras/blob/master/Commands.md#git-cp>.
+
+- Copy an existing file in a Git repo:
+
+`git cp {{file}} {{new_file}}`
```

Comment:
```suggestion
`git cp {{path/to/file}} {{path/to/new_file}}`
```

Reply:
No, because then the commands are just the same

Reply:
Oh, I see.